provisioning/control.sh

```diff
@@ -13,6 +9,12 @@ ssh-keyscan 192.168.33.15 >> /home/vagrant/.ssh/known_hosts
 ssh-keyscan 192.168.33.16 >> /home/vagrant/.ssh/known_hosts
 ssh-keyscan 192.168.33.17 >> /home/vagrant/.ssh/known_hosts
 
+ln -s /vagrant /home/vagrant/commcarehq-ansible
+mkdir /home/vagrant/commcare-hq-deploy
+sudo pip install virtualenv virtualenvwrapper
+sudo -H -u vagrant /vagrant/control/init.sh
```

Comment:
Vagrant control now uses the same script for ansible setup. The aliases don't make much sense though since vagrant doesn't have a proper `config` dir.